packages/core-mobile/app/services/notifications/news/unsubscribeForNews.ts

```diff
@@ -0,0 +1,47 @@
+import Logger from 'utils/Logger'
+import Config from 'react-native-config'
+import messaging from '@react-native-firebase/messaging'
+import fetchWithAppCheck from 'utils/httpClient'
+import { ChannelId } from '../channels'
+import { newsEvents } from './events'
+
+export async function unSubscribeForNews({
+  deviceArn,
+  channelIds
+}: {
+  deviceArn: string
+  channelIds?: ChannelId[]
```

Comment:
feel like it's better to be explicit and only accept a valid array here (either empty or not empty)